index.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const db = require('./data/db');  
+
+const server = express();
+
+server.listen(8000, () => console.log('API running...'));
```

Comment:
you have server listen running in 2 places.  probably only need it at the bottom.